src/components/_global/BalAccordion/BalAccordion.vue

```diff
@@ -34,7 +34,7 @@ const accordionHeightSetterElement = ref<HTMLElement>();
 const wrapperElement = ref<HTMLElement>();
 const handleBarElement = ref<HTMLElement>();
 const arrowElement = ref<HTMLElement>();
-const handleBarElements = ref<HTMLElement[]>([]);
+const handleBarElements = ref<(Element | ComponentPublicInstance)[]>([]);
```

Comment:
I can't say exactly. It could be another task to fix the accordion. Upgrading Vue broke the smooth opening animation, and accordion wouldn't close on click anymore.